docs/2-hooks/useCheckoutPricing.stories.mdx

```diff
@@ -0,0 +1,214 @@
+
+To initialize, call `useCheckoutPricing` [with an initial checkout pricing input][use-checkout-pricing-input].
+
+```js
+export default function PricingPreview() {
+  const initialPricingInput = {
+    subscriptions: [
+      {
+        plan: 'my-plan',
+      },
+    ],
+  };
+
+  const [{ price, loading }, setCheckoutPricing] = useCheckoutPricing(initialPricingInput);
+
+  if (!loading) {
+    return <div>{price.now.subtotal}</div>
+  }
+}
+```
+
+Calling `useCheckoutPricing` returns an array of two items:
+
+1. A [useCheckoutPricingState][use-checkout-pricing-state] object for reading output.
+2. A [setCheckoutPricing][set-checkout-pricing] function for updating input values.
+
+For complete documentation on what `useCheckoutPricing` returns, see the [useCheckoutPricingReturn reference page][use-checkout-pricing-return].
+
+## Initializing
+
+It's important to know that immediately after calling `useCheckoutPricing`, `useCheckoutPricingState.price` will default to an empty object while waiting for asynchronous actions to resolve.
+
+The below example will throw a type error because of this.
+```js
+export default function PricingPreview() {
+  const initialPricingInput = {
+    subscriptions: [
+      {
+        plan: 'my-plan',
+      },
+    ],
+  };
+
+  const [{ price }, setCheckoutPricing] = useCheckoutPricing(initialPricingInput);
+
+  return <div>{price.now.subtotal}</div>
+}
+```
+
+The [loading state](#loading-state) can be checked before rendering anything that uses `useCheckoutPricingState.price` to circumvent the error.
+
+```js
+export default function PricingPreview() {
+  const initialPricingInput = {
+    subscriptions: [
+      {
+        plan: 'my-plan',
+      },
+    ],
+  };
+
+  const [{ price, loading }, setCheckoutPricing] = useCheckoutPricing(initialPricingInput);
+
+  if (!loading) {
+    return <div>{price.now && price.now.subtotal}</div>
+  }
+}
+```
+
+Existence checking can be used as well.
+
+```js
+export default function PricingPreview() {
+  const initialPricingInput = {
+    subscriptions: [
+      {
+        plan: 'my-plan',
+      },
+    ],
+  };
+
+  const [{ price }, setCheckoutPricing] = useCheckoutPricing(initialPricingInput);
+
+  return <div>{price.now && price.now.subtotal}</div>
+}
+```
+
+## Updating
+
+The [setCheckoutPricing][set-checkout-pricing] function can be used to update the checkout pricing input if needed with a new [useCheckoutPricingInput][use-checkout-pricing-input].
+
+For example, if a customer wishes to choose a different plan for their subscription on a checkout page and the price preview needs to be updated, calling `setCheckoutPricing` with new inputs will update its associated `useCheckoutPricingState` and trigger a re-render.
+
+```js
+export default function PricingPreview() {
+  const [plan, setPlan] = useState("")
```

Comment:
lint nit: missing terminal semicolon

Reply:
+1, I have to clean up these code examples a bit